packages/webapp/public/locales/fr/disease.json

```diff
@@ -518,11 +518,11 @@
     "VIRUS": "Virus",
     "WEED": "Adventice"
   },
-  "PESTICIDES": {
```

Comment:
This is a great catch. Again not sure why i18n isn't generating a set of missing strings here.